Started by timer
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building in workspace /var/jenkins_home/workspace/fdc-prod/PHXB-Daily-Issue-Report-New
The recommended git tool is: NONE
using credential vinay-github
 > git rev-parse --resolve-git-dir /var/jenkins_home/workspace/fdc-prod/PHXB-Daily-Issue-Report-New/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/The-Frontdoor-Collective/fdc-etl.git # timeout=10
Fetching upstream changes from https://github.com/The-Frontdoor-Collective/fdc-etl.git
 > git --version # timeout=10
 > git --version # 'git version 2.30.2'
using GIT_ASKPASS to set credentials vinay-github
 > git fetch --tags --force --progress -- https://github.com/The-Frontdoor-Collective/fdc-etl.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/feature/ITDMF-126^{commit} # timeout=10
Checking out Revision a344f6b32ccee7507a08b1efac0985e1bd97eca6 (refs/remotes/origin/feature/ITDMF-126)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f a344f6b32ccee7507a08b1efac0985e1bd97eca6 # timeout=10
Commit message: "Update daily_exceptions_report_prod.py"
 > git rev-list --no-walk 49d1ad58f1880a3625a715cd44c51009724a8804 # timeout=10
[PHXB-Daily-Issue-Report-New] $ /bin/sh -xe /tmp/jenkins2604983177649673142.sh
+ cd python_scripts
+ python3 daily_exceptions_report_by_fdp.py mysql-prod.frontdoorcorp.net etluser fl@tCurve17 fdc_stage 15
python3: can't open file '/var/jenkins_home/workspace/fdc-prod/PHXB-Daily-Issue-Report-New/python_scripts/daily_exceptions_report_by_fdp.py': [Errno 2] No such file or directory
Build step 'Execute shell' marked build as failure
Finished: FAILURE
